Source/WebCore/page/scrolling/AsyncScrollingCoordinator.cpp

```diff
@@ -302,6 +302,22 @@ bool AsyncScrollingCoordinator::requestStopKeyboardScrollAnimation(ScrollableAre
     return true;
 }
 
+void AsyncScrollingCoordinator::setMouseIsInScrollbar(Scrollbar* scrollbar, bool didEnterScrollbar)
+{
+    ASSERT(isMainThread());
+    ASSERT(m_page);
+    auto scrollingNodeID = scrollbar->scrollableArea().scrollingNodeID();
+    if (!scrollingNodeID)
+        return;
+
+    auto stateNode = dynamicDowncast<ScrollingStateScrollingNode>(m_scrollingStateTree->stateNodeForID(scrollingNodeID));
+    if (!stateNode)
+        return;
+
+    stateNode->setMouseIsInScrollbar({ scrollbar->orientation() == ScrollbarOrientation::Vertical ? didEnterScrollbar : false, scrollbar->orientation() == ScrollbarOrientation::Vertical ? false : didEnterScrollbar });
+    scrollingStateTreePropertiesChanged();
```

Comment:
This shouldn’t be necessary. `ScrollingStateTree::setHasChangedProperties()` calls this.

Reply:
Removed.